### PR TITLE
ASM-7450 Increase puppet config timeout

### PR DESCRIPTION
**Pull request by sushilrai**
Updated timeout to 10 minutes while installing puppet agent on Server / VM. This will ensure we have correct timeout value on all supported operating system

### CODE REVIEW

File: tasks/windows2008.task/default-unattended.xml.erb [hunk 1]
```diff
@@ -98,6 +98,11 @@
         </RunSynchronousCommand>"
           end
         %>
+        <RunSynchronousCommand wcm:action="add">
+          <Description>Configure Puppet Agent config timeout</Description>
+          <Order>6</Order>
+          <Path>puppet agent set configtimeout 600</Path>
```
Comment (ripienaar) [inline tasks/windows2008.task/default-unattended.xml.erb]: hmm, what version agent is on windows?

Comment (sushilrai) [inline tasks/windows2008.task/default-unattended.xml.erb]: Windows client versions are updated . Its 3.6

Comment (ripienaar) [inline tasks/windows2008.task/default-unattended.xml.erb]: Weird, this `puppet agent set` worked only for a short time and later for replaced with `puppet config set`, would be safer to use that.  I think 3.6 should support it appears to have come in 3.5.

https://docs.puppet.com/pe/3.2/config_set.html

Comment (ripienaar) [inline tasks/windows2008.task/default-unattended.xml.erb]: actually `puppet agent set ...` just doesnt work, it starts the agent waiting for a cert, def needs to be `puppet config set`

File: tasks/windows2008.task/default-unattended.xml.erb [hunk 2]
```diff
@@ -101,7 +101,7 @@
         <RunSynchronousCommand wcm:action="add">
           <Description>Configure Puppet Agent config timeout</Description>
           <Order>6</Order>
-          <Path>puppet agent set configtimeout 600</Path>
+          <Path>puppet config set configtimeout 600</Path>
```
Comment (ripienaar) [inline tasks/windows2008.task/default-unattended.xml.erb]: think you need to supply --section, check the doc I linked

**Comment by aimers1975** [comment]
@sushilrai Can you address this review feedback if you want this merged?

**Comment by sushilrai** [comment]
Please dont merge it. timeout value is not updating if command is executed from the unattend.xml. Working on other options.

**Comment by sushilrai** [comment]
@ripienaar please review. Tested changes with Windows 2012, 2008R2 and HyperV

**Comment by ripienaar** [comment]
@sushilrai ah this got a lot more complex, i dont know anything about this stuff - the puppet command though I can confirm now works.  Can you think of anyone else who can review this?

**Comment by sushilrai** [comment]
@aimers1975 @gavin-scott  can you please take a look. We can discuss the changes if any clarification is required

**Pull request merged**